thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/DetectionOnboardJob.java

```diff
@@ -0,0 +1,39 @@
+package com.linkedin.thirdeye.anomaly.onboard;
+
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.configuration.Configuration;
+
+public interface DetectionOnboardJob {
+
+  /**
+   * Returns the unique name of this job.
+   * @return the unique name of this job.
+   */
+  String getName();
+
+  /**
+   * Initialize this job with the given properties.
+   *
+   * @param properties the properties to initialize the job.
+   */
+  void initialize(Map<String, String> properties);
```

Comment:
this "should" be the constructor. IMO avoid init() methods whenever you can. Pinot just underwent a major cleanup for this

Reply:
Fixed.